src/DIPS.Xamarin.UI/Controls/Slidable/SlidableLayout.cs

```diff
@@ -221,8 +221,9 @@ protected virtual void OnScrolled(double index)
         public static readonly BindableProperty ConfigProperty = BindableProperty.Create(
             nameof(Config),
             typeof(SliderConfig),
-            typeof(SlidableLayout));
-
+            typeof(SlidableLayout),
+            defaultValue: new SliderConfig(int.MinValue, int.MaxValue));
```

Comment:
Keep default to 0, 1
Else we might end up with insane values if fallback is used.